spec/models/user_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  let!(:external_location) { FactoryGirl.create(:external_location) }
+
+  # Columns
+  it { is_expected.to have_db_column(:name).of_type(:text)
+                        .with_options(null: false) }
+  it { is_expected.to have_db_column(:msf_location_id).of_type(:integer) }
+  it { is_expected.to have_db_column(:external_location).of_type(:text) }
+  it { is_expected.to have_db_column(:role).of_type(:enum)
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.